packages/components/src/notice/stories/index.js

```diff
@@ -1,74 +1,46 @@
-/**
- * External dependencies
- */
-import { boolean, select, text } from '@storybook/addon-knobs';
-
 /**
  * Internal dependencies
  */
 import Notice from '../';
 
+// TODO: Add a story involving NoticeList to help people understand
+// the difference between onDismiss/onRemove.
```

Comment:
[Similar situation to `Snackbar` and `SnackbarList`](https://github.com/WordPress/gutenberg/pull/45472#pullrequestreview-1166742669). In those situations, should we treat these components as part of the same family? In that case, which component is the "main" component?

Reply:
I think we'd need to evaluate each case based on whether the `Foo` component can be used independently of the `FooList` or `FooGroup`. If it can be used independently, that would be the main component.